Sources/Recycling/GridViews/Ads/GridView/AdsGridView.swift

```diff
@@ -229,6 +244,12 @@ extension AdsGridView: AdsGridViewLayoutDelegate {
             return defaultImageSize.height / defaultImageSize.width
         }
 
+        var height = model.imageSize.height
+        // We want large ads to show in a more prominent way,
+        // hence the change in image height ratio.
+        if model.isLargeAd {
+            height *= 1.8
```

Comment:
I guess `height` should either be used or this code removed if it is no longer needed